test/test_tensor_creation_ops.py

```diff
@@ -1968,11 +1968,12 @@ def test_empty_full(self, device):
         torch_device = torch.device(device)
         device_type = torch_device.type
 
+        dtypes = get_all_dtypes(include_half=False, include_bfloat16=False, include_complex32=True)
         if device_type == 'cpu':
-            do_test_empty_full(self, get_all_math_dtypes('cpu'), torch.strided, torch_device)
+            do_test_empty_full(self, dtypes, torch.strided, torch_device)
         if device_type == 'cuda':
-            do_test_empty_full(self, get_all_math_dtypes('cpu'), torch.strided, None)
```

Comment:
Any idea if this was intended to be `cpu` even when `device_type` is `cuda`

cc: @mruberry

Reply:
Ref PR: https://github.com/pytorch/pytorch/pull/43104/files

Reply:
When refactoring our test suite I often did the minimum to get the tests to work in the new framework (it was a lot of LoC!), so I probably just didn't update this test completely. It doesn't even use the @dtypes decorator!

(or maybe the @dtypes decorator didn't even exist at the time)